python/guillotina/requirements.txt

```diff
@@ -0,0 +1,4 @@
+guillotina==6.0.18
```

Comment:
could you use a more relax constraint (a range from 6.0 to 6.1) ?

Reply:
done!